packages/frontend/src/translations/en.global.json

```diff
@@ -1657,7 +1657,7 @@
     "twoFactor": {
         "action": {
             "addKey": {
-                "full": "WARNING: Entering the code below will authorize full access to your NEAR account: ${accountId}. If you did not initiate this action, please DO NOT continue.<br /><br />This should only be done if you are adding a new full access key to your account. In all other cases, this is very dangerous.<br /><br /> The public key you are adding is: ${publicKey} ",
+                "full": "Entering the code below will authorize full access to your NEAR account: ${accountId} If you did not initiate this action, please DO NOT continue.",
```

Comment:
for 1 account, text should read `account needs`
for more than 1 account,  should read `accounts need`

Reply:
you mean at `twoFactorDisbleBanner.title`? it is already been handled